data/fixtures/scopes/r/argument.formal.iteration.scope

```diff
@@ -0,0 +1,17 @@
+abc <- function(x, y){}
+
+---
+
+[#1 Range] = 0:16-0:20
+                  >----<
+0| abc <- function(x, y){}
+
+[#1 Domain] = 0:15-0:21
+                 >------<
+0| abc <- function(x, y){}
```

Comment:
That domain should include the entire function

eg
https://github.com/cursorless-dev/cursorless/blob/97bf1d6651000087ea1762aec9bb5ca50bc1b00e/data/fixtures/scopes/javascript.core/argument.formal.iteration.scope

Reply:
Have you read my response about that?
https://github.com/cursorless-dev/cursorless/pull/2721/files#r2106136228

Reply:
You probably want to include the function name as well so that you can take all arguments from the function name